frontend/src/app/offers/offers.model.ts

```diff
@@ -0,0 +1,11 @@
+export class Offer {
+    id: number;
+    image: string;
+    location: string;
+    organization: string;
+    slug: string;
+    startedAt: string | null;
```

Comment:
I believe, `| null` is not necessary here.

Reply:
This is possible value, when Offer.startedAt is null we print "teraz"